VCFv4.3.tex

```diff
@@ -331,7 +331,7 @@ \subsubsection{Fixed fields}
   \item INFO - additional information: (String, no semi-colons or equals-signs permitted; commas are permitted only as delimiters for lists of
   values; characters with special meaning can be encoded using the percent encoding, see Section~\ref{character-encoding}; space characters are allowed)
   INFO fields are encoded as a semicolon-separated series of short keys with optional values in the format: $<$key$>$=$<$data$>$[,data].
-  INFO keys must match the regular expression \texttt{\^{}[A-Za-z\_][0-9A-Za-z\_.]*\$}, duplicate fields are not allowed. Arbitrary keys are permitted, although the sub-fields listed in Table~\ref{table:reserved-info} are reserved (albeit optional).
+  INFO keys must match the regular expression \texttt{\^{}[A-Za-z\_][0-9A-Za-z\_.]*\$}, although ``1000G'' is allowed as a special legacy value. Duplicate fields are not allowed. Arbitrary keys are permitted, although the sub-fields listed in Table~\ref{table:reserved-info} are reserved (albeit optional).
```

Comment:
It might be best to add 1000G as an or clause in the regex.
Something like: `^([A-Za-z ][0-9A-Za-z .]*|1000G)$`
Then I would still include the note that 1000G is included as a legacy value.  It would let people use the regex to validate without any caveats which seems nice.

If it's kept this way, I would probably change the wording slightly.   Maybe "..., except for the key "1000G" which is allowed as a special legacy value."